generated/nirfmxwlan/nirfmxwlan.proto

```diff
@@ -31,27 +590,3421 @@ enum Boolean {
+  SEM_LOWER_OFFSET_MEASUREMENT_STATUS_FAIL = 0;
+  SEM_LOWER_OFFSET_MEASUREMENT_STATUS_PASS = 1;
+}
+
+enum SemMaskType {
+  option allow_alias = true;
+  SEM_MASK_TYPE_UNSPECIFIED = 0;
+  SEM_MASK_TYPE_STANDARD = 0;
+  SEM_MASK_TYPE_CUSTOM = 1;
+}
+
+enum SemMeasurementStatus {
+  option allow_alias = true;
+  SEM_MEASUREMENT_STATUS_UNSPECIFIED = 0;
+  SEM_MEASUREMENT_STATUS_FAIL = 0;
+  SEM_MEASUREMENT_STATUS_PASS = 1;
+}
+
+enum SemSpanAuto {
+  option allow_alias = true;
+  SEM_SPAN_AUTO_UNSPECIFIED = 0;
+  SEM_SPAN_AUTO_FALSE = 0;
+  SEM_SPAN_AUTO_TRUE = 1;
+}
+
+enum SemSweepTimeAuto {
+  option allow_alias = true;
+  SEM_SWEEP_TIME_AUTO_UNSPECIFIED = 0;
+  SEM_SWEEP_TIME_AUTO_FALSE = 0;
+  SEM_SWEEP_TIME_AUTO_TRUE = 1;
+}
+
+enum SemUpperOffsetMeasurementStatus {
+  option allow_alias = true;
+  SEM_UPPER_OFFSET_MEASUREMENT_STATUS_UNSPECIFIED = 0;
+  SEM_UPPER_OFFSET_MEASUREMENT_STATUS_FAIL = 0;
+  SEM_UPPER_OFFSET_MEASUREMENT_STATUS_PASS = 1;
+}
+
+enum Standard {
+  option allow_alias = true;
+  STANDARD_UNSPECIFIED = 0;
+  STANDARD_802_11_AG = 0;
+  STANDARD_802_11_B = 1;
+  STANDARD_802_11_J = 2;
+  STANDARD_802_11_P = 3;
+  STANDARD_802_11_N = 4;
+  STANDARD_802_11_AC = 5;
+  STANDARD_802_11_AX = 6;
+  STANDARD_802_11_BE = 7;
+  STANDARD_UNKNOWN = -1;
+}
+
+enum TriggerMinimumQuietTimeMode {
+  option allow_alias = true;
+  TRIGGER_MINIMUM_QUIET_TIME_MODE_UNSPECIFIED = 0;
+  TRIGGER_MINIMUM_QUIET_TIME_MODE_MANUAL = 0;
+  TRIGGER_MINIMUM_QUIET_TIME_MODE_AUTO = 1;
+}
+
+enum TxpAveragingEnabled {
+  option allow_alias = true;
+  TXP_AVERAGING_ENABLED_UNSPECIFIED = 0;
+  TXP_AVERAGING_ENABLED_FALSE = 0;
+  TXP_AVERAGING_ENABLED_TRUE = 1;
+}
+
+enum TxpBurstDetectionEnabled {
+  option allow_alias = true;
+  TXP_BURST_DETECTION_ENABLED_UNSPECIFIED = 0;
+  TXP_BURST_DETECTION_ENABLED_FALSE = 0;
+  TXP_BURST_DETECTION_ENABLED_TRUE = 1;
+}
+
+enum NiRFmxWLANInt32AttributeValues {
+  option allow_alias = true;
+  NIRFMXWLAN_INT32_UNSPECIFIED = 0;
+  NIRFMXWLAN_INT32_DIGITAL_EDGE_TRIGGER_EDGE_RISING_EDGE = 0;
+  NIRFMXWLAN_INT32_DIGITAL_EDGE_TRIGGER_EDGE_FALLING_EDGE = 1;
+  NIRFMXWLAN_INT32_DSSS_MOD_ACC_ACQUISITION_LENGTH_MODE_DSSSMODACC_ACQUISITION_LENGTH_MODE_MANUAL = 0;
+  NIRFMXWLAN_INT32_DSSS_MOD_ACC_ACQUISITION_LENGTH_MODE_DSSSMODACC_ACQUISITION_LENGTH_MODE_AUTO = 1;
+  NIRFMXWLAN_INT32_DSSS_MOD_ACC_AVERAGING_ENABLED_FALSE = 0;
+  NIRFMXWLAN_INT32_DSSS_MOD_ACC_AVERAGING_ENABLED_TRUE = 1;
+  NIRFMXWLAN_INT32_DSSS_MOD_ACC_BURST_START_DETECTION_ENABLED_FALSE = 0;
+  NIRFMXWLAN_INT32_DSSS_MOD_ACC_BURST_START_DETECTION_ENABLED_TRUE = 1;
+  NIRFMXWLAN_INT32_DSSS_MOD_ACC_CHIP_CLOCK_ERROR_CORRECTION_ENABLED_FALSE = 0;
+  NIRFMXWLAN_INT32_DSSS_MOD_ACC_CHIP_CLOCK_ERROR_CORRECTION_ENABLED_TRUE = 1;
+  NIRFMXWLAN_INT32_DSSS_MOD_ACC_DATA_DECODING_ENABLED_FALSE = 0;
+  NIRFMXWLAN_INT32_DSSS_MOD_ACC_DATA_DECODING_ENABLED_TRUE = 1;
+  NIRFMXWLAN_INT32_DSSS_MOD_ACC_DATA_MODULATION_FORMAT_DSSSMODACC_DATA_MODULATION_FORMAT_DSSS1_MBPS = 0;
```

Comment:
Note: not part of this change (should be added as a task either on this story or the api review).

These are iffy. 

I.e., 

RFMXWLAN_VAL_DSSSMODACC_DATA_MODULATION_FORMAT_DSSS1MBPS

`NIRFMXWLAN_INT32_DSSS_MOD_ACC_DATA_MODULATION_FORMAT_DSSSMODACC_DATA_MODULATION_FORMAT_DSSS1_MBPS `

I'm concerned that the C API does use `_`s in some of these, but not the same ones we have.

I'm not sure where we're introducing the extra `_`s relative to the C API. If it's in RFmx metadata it's OK. If it's in one of our identifier conversion methods then that's not great.

Here's the full block of defines from the C API:

```c
// Values for RFMXWLAN_ATTR_DSSSMODACC_RESULTS_DATA_MODULATION_FORMAT
#define RFMXWLAN_VAL_DSSSMODACC_DATA_MODULATION_FORMAT_DSSS1MBPS                                    0
#define RFMXWLAN_VAL_DSSSMODACC_DATA_MODULATION_FORMAT_DSSS2MBPS                                    1
#define RFMXWLAN_VAL_DSSSMODACC_DATA_MODULATION_FORMAT_CCK5_5MBPS                                   2
#define RFMXWLAN_VAL_DSSSMODACC_DATA_MODULATION_FORMAT_CCK11MBPS                                    3
#define RFMXWLAN_VAL_DSSSMODACC_DATA_MODULATION_FORMAT_PBCC5_5MBPS                                  4
#define RFMXWLAN_VAL_DSSSMODACC_DATA_MODULATION_FORMAT_PBCC11MBPS                                   5
#define RFMXWLAN_VAL_DSSSMODACC_DATA_MODULATION_FORMAT_PBCC22MBPS                                   6
#define RFMXWLAN_VAL_DSSSMODACC_DATA_MODULATION_FORMAT_PBCC33MBPS                                   7
```

Reply:
Just to tack on to this. I think we'll need to consider this same scenario for some of the DSSSMODACC defines/enum values as well .

Reply:
@DavidCurtiss and I discussed this problem (and the related one for param names) and he's started looking at scripts for fixing this stuff.

We are thinking that we want to use `DSSS_MOD_ACC` and equivalent everywhere instead of `DSSSMODACC`. This will be more consistent with enum names and function names. And also just-plain-better. We'll have to see how that works in the implementation. Some stuff will naturally work that way (like the grpc-device code that adds enum value prefixes) and some might need to be corrected (attribute names?) but we'll have some options on how to do that. For example there are some PascalCaseNames in the related metadata: 
![image](https://user-images.githubusercontent.com/56981669/151578633-dd8e8fe6-ed48-405f-bc3b-aa2d1e4e9587.png)

And we'll be building up a list of back-to-back acronym tokens either way.